testing/trino-product-tests/pom.xml

```diff
@@ -340,6 +340,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <configuration>
+                    <!-- required to inject a custom redirection handler into the JDBC driver -->
+                    <excludes>**/TestingRedirectHandlerInjector.java</excludes>
```

Comment:
I don't believe the exclusion is correct.
The TestingRedirectHandlerInjector class simply resides in a wrong folder.

Reply:
it's a test code's hook, not API

Reply:
Should I move it to `client/trino-jdbc/src/main` or `client/trino-jdbc/src/test`?

Reply:
Oooh 🤦